City-of-Helsinki/drupal-asuntotuotanto
# Added a functionality to add custom icons on custom theme.

How to test: 
- Checkout this branch
- Open terminal and go to `/public/themes/custom/asuntotuotanto`
   - Run `nvm use && npm i && npm run build`
   - Check that the build process creates an icon sprite to `/public/themes/custom/asuntotuotanto/dist/icons/sprite.svg`
- Try to insert an icon with the twig like so:
```
{# Asuntotuotanto specific icons #}
{% include "@asuntotuotanto/misc/icon.twig" with {icon: 'testing'} %}
{# HDBT specific icons #}
{% include "@hdbt/misc/icon.twig" with {icon: 'google-view'} %}
```
The actual svg files lives in `/public/themes/custom/asuntotuotanto/src/icons/`
